client/src/components/modal/SearchModal.jsx

```
@@ -0,0 +1,46 @@
+  return (
+    <div className={styles.modal}>
+      <div className={styles.closeButtonWrapper}>
+        <img
+          src={closeButton}
+          alt="closeButton"
+          className={styles.closeButton}
+          onClick={searchHandler}
+        />
+      </div>
+      <div className={styles.searchBoardWrapper}>
+        <SearchBoard clickHandler={clickHandler}></SearchBoard>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)